test/functional_cpu_test.py

```diff
@@ -273,7 +273,9 @@ def test_linearity_of_istft4(self):
         self._test_linearity_of_istft(data_size, kwargs4, atol=1e-5, rtol=1e-8)
 
 
-class TestDetectPitchFrequency(unittest.TestCase):
+class TestDetectPitchFrequency(common_utils.TorchaudioTestCase):
+    backend = 'default'
```

Comment:
nit: this is the only one that needs a backend to be specified?

Reply:
there are couple of them, the less class requires backend, the better. yet still the backend is reset for the each class.